tools/wptrunner/wptrunner/wptcommandline.py

```diff
@@ -539,7 +539,8 @@ def check_args(kwargs):
 
     if kwargs['extra_prefs']:
         # If a single pref is passed in as a string, make it a list
-        if type(kwargs['extra_prefs']) in (str, unicode):
+        from six import text_type
```

Comment:
Can we have this import at the top of the file?

Reply:
sure